lib/menu.rb

```diff
@@ -0,0 +1,24 @@
+class Menu
+  attr_reader :dishes
+
+  def initialize(dishes = default_dishes)
+    @dishes = dishes
+  end
+
+  def default_dishes
+    dishes = []
+    dishes << Dish.new("Curry", 8)
+    dishes << Dish.new("Rice", 3)
+  end
+
+  def print
+    puts "Menu:"
+    @dishes.each { |dish| print_dish(dish) }
+  end
```

Comment:
Can this be added to the printer class? I'm not entirely sure that it should, but given that all other 'print' methods are, it seems like this should be as well.